finalPrep/annotations.js

```diff
@@ -2,10 +2,16 @@
 // Use the double // to create a new comment
```

Comment:
Really nice work throughout this file - you have a solid understanding of the utility of each line of code and are using technical vocabulary accurately!